tests/Nelmio/Alice/Fixtures/Builder/Reference.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *  
+ * (c) Nelmio <hello@nelm.io>
+ *  
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nelmio\Alice\Fixtures\Builder\Methods;
+
+use Nelmio\Alice\Fixtures\Fixture;
+
+/**
+ * List a list of fixture reference, with the expected result. The list is flagged to be able to easily extract some
+ * parts of it. This class aims at centralising all the possible fixture tested to avoid too much duplication.
+ */
+final class Reference
```

Comment:
@theofidry I like the decisions you've made and the tests you're running - I'm personally not in favor of using engineered solutions in tests (factories, etc.) since it makes things a bit more opaque, but that's just my personal taste. My preference is to have tests be stupidly obvious, because from my perspective, tests are the best documentation there is for a library.

Other than that, I like it all.